x-pack/plugins/alerting/server/task_runner/task_runner.ts

```diff
@@ -494,6 +497,7 @@ export class TaskRunner<
 
     const namespace = this.context.spaceIdToNamespace(spaceId);
     const eventLogger = this.context.eventLogger;
+    const scheduleDelay = Date.now() - this.taskInstance.runAt.getTime();
```

Comment:
nit: Maybe use `runDate` so that the logged start time will match exactly with the calculated schedule delay?